decisions/0006-program-slugs-in-publisher.rst

```diff
@@ -0,0 +1,30 @@
+6. Program Slugs in Publisher
+-------------------------------
+
+Status
+------
+
+Accepted (July 2023)
+
+Context
+-------
+
+Ability to update the slugs for programs is needed to be accessible to a group of users.
+These slugs are used in generating marketing url for each type of programs.
+These program slugs will be programmatically generated, just like courses.
+In order to grant the ability to update slugs for programs, we will going to display programs on Publisher due to it's ease of access control.
+Instead of displaying all program information on the publisher, we will only show the editable slug field.
+
+Decision
+--------
+
+A new tab will be added on Publisher named "Programs/Degrees" in the header.
+Upon clicking the tab, the user will be shown a list of programs with minimal info.
+Each item in the list will take the user to it's detail page where an input field will be available for slugs and a save button.
```

Comment:
- it's --> its
- I dont think we expect to have a detail page. It can be a simple pop-up. Using the word "details" can give an incorrect impression.

Reply:
Are we going to use a modal or a simple click and edit field on the programs list page?

Reply:
Wouldn't it be easier here? We know program/degree are not going to come into publisher anytime soon.

Reply:
Okay I'm adding that a modal will be displayed instead of detail page.